src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/HearingDetailsController.java

```diff
@@ -76,6 +79,26 @@ public ResponseEntity<CCDCallbackResponse> handleListingSelected(
         return getCallbackRespEntityNoErrors(caseData);
     }
 
+    @PostMapping(value = "/hearingMidEventValidation", consumes = APPLICATION_JSON_VALUE)
+    @Operation(summary = "validates the hearing number and the hearing days to prevent their creation.")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "Accessed successfully"),
+            @ApiResponse(responseCode = "400", description = "Bad Request"),
+            @ApiResponse(responseCode = "500", description = "Internal Server Error")
+    })
+    public ResponseEntity<CCDCallbackResponse> hearingMidEventValidation(
+            @RequestBody CCDRequest ccdRequest,
+            @RequestHeader(value = "Authorization") String userToken) {
+        if (!verifyTokenService.verifyTokenSignature(userToken)) {
+            log.error(INVALID_TOKEN, userToken);
+            return ResponseEntity.status(FORBIDDEN.value()).build();
+        }
+
+        var caseData = ccdRequest.getCaseDetails().getCaseData();
```

Comment:
Prefer not to use var.
Worth applying across this PR

Reply:
replaced vars.